tools/java/java-build/src/com/google/i18n/phonenumbers/BuildMetadataProtoFromXml.java

```diff
@@ -133,7 +140,7 @@ public boolean start() {
     }
 
     if (inputFile == null ||
-        outputDir == null ||
+        ((outputDir == null) == (outputFile == null)) ||
```

Comment:
Discussed in person: this is supposed to check that one and only one of them is non-null.